src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -457,4 +457,40 @@ public Keys call() throws SaltStackException {
     public EventStream events() {
         return new EventStream(config);
     }
+
+    /**
+     * Trigger an event in Salt with the specified tag and data.
+     *
+     * POST /hook
+     *
+     * @param eventTag the event tag
+     * @param eventData the event data. Must be valid JSON.
+     * @return the boolean value returned by Salt. If true the event was triggered successfully. A value of false
+     * is returned only if Salt itself returns false; it does not mean a communication failure.
+     * @throws SaltStackException if anything goes wrong
+     */
+    public boolean hook(String eventTag, String eventData) throws SaltStackException {
+        String tag = eventTag != null ? eventTag : "";
+        Map<String, Object> result = connectionFactory
+                .create("/hook/" + tag, JsonParser.MAP, config)
+                .getResult(eventData);
+        return Boolean.TRUE.equals(result.get("success"));
+    }
+
+    /**
+     * Asynchronously trigger an event in Salt with the specified tag and data.
+     *
+     * POST /hook
+     *
+     * @return Future containing a boolean value indicating the success of failure of the event triggering .
```

Comment:
Sorry to bother you again with this, but please describe the parameters here as well.